android-app/app/src/main/java/de/dhbw/ka/se/fibo/ui/adding/AddingFragment.java

```diff
@@ -139,7 +141,7 @@ private void initializeButtons() {
                 Toast.makeText(requireContext(), "Some required inputs are empty or wrong formatted", Toast.LENGTH_SHORT).show();
             } else {
                 ApplicationState.getInstance(requireContext()).addCashflow(newCashFlow);
-                System.out.println(newCashFlow);
+                System.out.println(JSONObject.wrap(newCashFlow));
```

Comment:
should be a `Log.i()` call

Reply:
Its a old experiment that should be removed. It does not work as expected :)